index.d.ts

```diff
@@ -25,20 +17,20 @@ declare namespace ReactMarkdown {
     readonly indent?: number[]
   }
 
-  export type NodeType = Content['type']
+  type NodeType = Content['type']
 
-  export type AlignType = 'left' | 'right' | 'center' | null
+  type AlignType = 'left' | 'right' | 'center' | null
```

Comment:
Can these be imported (and `ReferecenType`)? https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f37828cb0a0d60300a749c2d3f12d9d4314a59c2/types/mdast/index.d.ts#L9